### PR TITLE
[MPT-120] Migrate from pages to app router

### PR DESCRIPTION
**Pull request by dabby9734**

### CODE REVIEW

**Comment by wei2912** [comment]
Production build seems to work fine, but I'm unable to get `npm run dev` (or `next dev`) working properly. The mentors page doesn't render:

image

File: src/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  manifest: "/manifest.json",
+
+  // TODO: Add Google metadata
+  // The following needs to be added to the HTML <head> somehow:
+  // <meta itemProp="name" content="Advisory Mentorship Programme" />
+  // <meta
+  //     itemProp="description"
+  //     content="The Advisory Mentorship Programme"
+  // />
+  // <meta itemProp="image" content="/advisory-mentorship-logo.jpg" />
```
Comment (wei2912) [inline src/app/layout.tsx:53]: To address later on.

File: src/app/faq/page.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+"use client";
```
Comment (wei2912) [inline src/app/faq/page.tsx:1]: To check - was there any reason why this isn't added to the other pages?

**Review by wei2912 — CHANGES_REQUESTED** [review]
Generally looks okay, but need to address Google metadata issue and above `npm run dev`. Will look into the former after @dabby9734 has clarified on the latter issue.

**Comment by wei2912** [comment]
Have checked the `npm run dev` issue, doesn't seem to be present in my new dev env - might have forgotten to run `npm i`...

Please rebase to fix the `package.json` conflicts too!

File: src/app/layout.tsx (hunk 1 above)
Comment (dabby9734) [reply src/app/layout.tsx]: addressed in 64480e00d11c839d0e98c0a6545a927ae1f3eae6

File: src/app/faq/page.tsx (hunk 2 above)
Comment (dabby9734) [reply src/app/faq/page.tsx:1]: All the pages have this (see src/app/faq/page.tsx, src/app/mentors/page.tsx, src/app/page.tsx)

**Pull request merged**